app/views/chatrooms/_member_list.html.erb

```diff
@@ -0,0 +1,17 @@
+<% @member.each do |member| %>

```

Comment:
`render partials with collection` mà em
viết như này không khác gì cái chị comment trước cả

Reply:
Chỗ này không hiểu sao mà em render @member nó toàn báo missing template